integration/test_resources/help/cloud-account_iac-generate_gcp

```diff
@@ -0,0 +1,55 @@
+Use this command to generate Terraform code for deploying Lacework into an GCP environment.
+
+By default, this command will function interactively, prompting for the required information to setup the new cloud account. In interactive mode, this command will:
```

Comment:
How about: this command interactively prompts for the required information to...